int64 id
ltlplanner/ROI_node[] nodes
ltlplanner/ROI_edge[] edges
ltlplanner/ROI_node initial_node
